plugins/module_utils/network/sonic/config/pki/pki.py

```diff
@@ -44,10 +44,13 @@
 SECURITY_PROFILE_PATH = (
     "data/openconfig-pki:pki/security-profiles/security-profile"
 )
+INSTALL_PATH = "operations/openconfig-pki-rpc:crypto-host-cert-install"
+DELETE_PATH = "operations/openconfig-pki-rpc:crypto-host-cert-delete"
 
 PATCH = "patch"
 DELETE = "delete"
 PUT = "put"
+POST = 'post'
```

Comment:
```suggestion
POST = "post"
```
A little nitpick but for consistency you should use double quotes here